### PR TITLE
Feature/mortar 76

### PR DESCRIPTION
**Pull request by vamidjo**
https://jira.nationalgeographic.com/browse/MORTAR-76

git checkout
gulp serve
http://localhost:4000/patterns

### CODE REVIEW

File: app/styles/blocks/_counters.scss [hunk 1]
```diff
@@ -1,11 +1,20 @@
+.mt_numericcounter {
```
Comment (xiwcx) [inline app/styles/blocks/_counters.scss]: did you try the generic class for this first? if so, proceed. otherwise, let's give it a shot.

Comment (vamidjo) [inline app/styles/blocks/_counters.scss]: What's the generic class?

Comment (vamidjo) [inline app/styles/blocks/_counters.scss]: I tried 

```
<button class="iconandlabel--horizontal mt_numericcounter-button">
<svg class="mt_icon">...</svg>
</button>
```
but this only accounts for single pairing of text + icons and everything else is displayed as a block element.

It looks likes this 

< 1
/
12 >

Comment (xiwcx) [inline app/styles/blocks/_counters.scss]: did you try `<div class="mt_numericcounter iconandlabel--horizontal">`?

Comment (vamidjo) [inline app/styles/blocks/_counters.scss]: Yes, no luck

Comment (xiwcx) [inline app/styles/blocks/_counters.scss]: great, consider this merged!

**Pull request merged**